scripts/options.js

```diff
@@ -71,6 +71,7 @@ document.getElementById('no-kk').onclick = saveOptions;
 document.getElementById('enable-kk').onclick = saveOptions;
 document.getElementById('always-kk').onclick = saveOptions;
 document.getElementById('enable-notifications').onclick = saveOptions;
+document.getElementById('enable-town-tune').onclick = saveOptions;
```

Comment:
This line is necessary to trigger new option retrieval when the town tune is enabled or disabled. I couldn't find any reason why it wouldn't be here, so I assume it was just an accidental omission.